apps-infra/infra-tools/__main__.py

```diff
@@ -0,0 +1,35 @@
+from pulumi_kubernetes.helm.v3 import Chart, ChartOpts, FetchOpts
```

Comment:
You can keep all these tooling deployments including istio, argo, prometheus, grafana, keptn, ... under one place: infra tools or add-ons but not both.